src/PageContainer/component.js

```diff
@@ -100,8 +100,10 @@ export default class PageContainer extends Component {
     }
 
     const item = find(context.collection, pageUrl)
-
+    console.log(isClient)
+    console.log(typeof window !== "undefined")
```

Comment:
You define window after importing the file, and isClient is defined at the top of the file.